tests/mlio-test/test_text_line_reader.cxx

```diff
@@ -0,0 +1,113 @@
+#include <gtest/gtest.h>
+#include <mlio.h>
+#include <unistd.h>
+
+#include "../../src/mlio/detail/error.h"
```

Comment:
You should not include internal include files in your unit tests. If they have features that are necessary/useful for unit tests, we should evaluate our options about how we can refactor or copy them.

Reply:
by internal, are you only referring to `#include "../../src/mlio/detail/error.h"`?

Reply:
yes.